packages/web-react/src/components/Flex/demo/FlexResponsiveWrapping.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import DocsBox from '../../../../docs/DocsBox';
+import Flex from '../Flex';
+
+const FlexResponsiveWrapping = () => (
+  <Flex isWrapping={{ mobile: true, tablet: false }}>
+    {Array.from({ length: 15 }, (_, i) => (
```

Comment:
```suggestion
    {[...Array(15)].map((_, index) => {
```

this is what we do elsewhere 🤷